frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot-class-details.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import {
+  SectionHeading,
+  ResourceSummary,
+  navFactory,
+  Kebab,
+} from '@console/internal/components/utils';
+import { VolumeSnapshotClassKind } from '@console/internal/module/k8s';
+import { DetailsPage } from '@console/internal/components/factory';
+import { volumeSnapshotStatus } from '../../status';
+
+const { editYaml } = navFactory;
+
+const Details: React.FC<DetailsProps> = ({ obj }) => {
```

Comment:
nit: you can directly return the `<div className="co-m-pane__body">` here

Reply:
ack